partiql-types/src/main/kotlin/org/partiql/value/datetime/DateTime.kt

```diff
@@ -528,6 +533,17 @@ public abstract class TimestampWithTimeZone : Timestamp {
 
     public final override fun toString(): String =
         "${this.javaClass.simpleName}(year=$year, month=$month, day=$day, hour=$hour, minute=$minute, second=$decimalSecond, timeZone=$timeZone)"
+
+    public companion object {
+
+        @JvmStatic
+        public fun nowZ(): TimestampWithTimeZone {
+            val javaNowZ = OffsetDateTime.now(ZoneOffset.UTC)
+            val date = SqlDate.of(javaNowZ.year, javaNowZ.monthValue, javaNowZ.dayOfMonth)
+            val time = OffsetTimeLowPrecision.of(javaNowZ.hour, javaNowZ.minute, javaNowZ.second, javaNowZ.nano, TimeZone.UtcOffset.of(0))
+            return forDateTime(date, time)
+        }
```

Comment:
Why not just `OffsetLowPrecision.nowZ()`. https://github.com/partiql/partiql-lang-kotlin/blob/main/partiql-types/src/main/kotlin/org/partiql/value/datetime/impl/OffsetTimestampLowPrecision.kt#L136-L140